src/components/common/heading/common/helper/get-styles/get-styles.helper.ts

```diff
@@ -0,0 +1,30 @@
+import { StyleProp, TextStyle } from 'react-native'
```

Comment:
Put this helper into `components/heading/helpers/getStylesByType.ts`. Then export it as default in index.ts